config.js

```diff
@@ -29,6 +31,8 @@ const MU_AUTH_ALLOWED_GROUPS = [
     { "variables": [], "name": "kanselarij-write" },
     { "variables": [], "name": "sign-flow-read" },
     { "variables": [], "name": "sign-flow-write" },
+    { "variables": [], "name": "parliament-flow-read" },
+    { "variables": [], "name": "parliament-flow-write" },
```

Comment:
I was under the impression that secretarie would never write these, unless that changed while I was on vacation?